.drone.star

```diff
@@ -1200,7 +1200,7 @@ def installCore(version, db, useBundledApp):
 		'image': 'owncloudci/core',
 		'pull': 'always',
 		'settings': {
-			'version': version,
+			'git_reference': 'loginexception',
```

Comment:
This change will need to be removed after the core PR has been merged and the next nightly core tarball has been built. Then let CI run again here...

Reply:
The core pr has been merged and this change has been removed.